index.js

```diff
@@ -65,6 +65,12 @@ class Response {
     else if (CONNECTION.test(name)) this._keepAlive = false
   }
 
+  setHeaders (obj) {
+    Object.keys(obj).map(key => {
+      this.setHeader(key, obj[key])
+    })
```

Comment:
could we make this:

``` js
const names = Object.keys(obj)
for (var i = 0; i < names.length; i++) {
  const name = names[i]
  this.setHeader(name, obj[name])
}
```

then we save a function allocation

Reply:
or use:
```js
const names = Object.keys(obj)
names.forEach(key => {
  this.setHeader(key, obj[key])
})
```